flask_common/declenum.py

```diff
@@ -0,0 +1,89 @@
+# From: http://techspot.zzzeek.org/2011/01/14/the-enum-recipe/
+
+from sqlalchemy.types import SchemaType, TypeDecorator, Enum
+import re
+
+class DeclEnumType(SchemaType, TypeDecorator):
```

Comment:
Docstring on these classes or at least a higher level description at the top of how to use them in the basic cases